sdk/core/core-lro/README.md

```diff
@@ -67,42 +56,55 @@ https://github.com/Azure/azure-sdk-for-js/search?q="from+%40azure%5C%2Fcore-lro"
 
 ### Implementing an operation
 
-First, you'll need to import the definitions of a poll operation and its state, as follows:
+To work with a Long Running Operation, we need to define how to update, cancel
+and serialize the information related to the remote pending task.
+To implement this class for your library or application,
+you'll need to import the definitions of a poll operation and its state.
 
 ```typescript
 import { PollOperationState, PollOperation } from "@azure/core-lro";
 ```
 
-A **PollOperation** requires implementing its state. We recommend extending the PollOperationState we provide.
-An example can be:
+A `PollOperation` is an interface that defines how to update the local reference of the state
+of the remote long running operation, just as well as how to request the cancellation of the same operation.
+Besides updating and cancelling, it also defines how to serialize its information,
+so that polling can be resumed at any time from a previously serialized operation
+(more at [Serializing an operation](#serializing-an-operation)).
+
+`PollOperationState` provides a basic state for the poll operation.
+It contains an opinionated list of the smallest set of properties needed
```

Comment:
We can probably avoid adjectives like "opinionated". The list is the minimal set for LRO to work well, which seems okay.